tools/packaging/packaging.psm1

```diff
@@ -20,7 +20,7 @@ function Start-PSPackage {
         [string]$Name = "powershell",
 
         # Ubuntu, CentOS, Fedora, macOS, and Windows packages are supported
-        [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg", "tar")]
+        [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg", "tar", "tar-arm")]
```

Comment:
There are comments about `deb-arm` right below the parameter block, please update or remove them if they are not applicable anymore.

Reply:
Will remove as you can only build a deb package by running this cmdlet on the actual platform itself.